clerk/instance_organization_permissions.go

```diff
@@ -4,7 +4,7 @@ import (
 	"net/http"
 )
 
-type OrganizationPermission struct {
+type Permission struct {
```

Comment:
:question: Are we supposed to move these to different files as well?

Reply:
Sure thing, moved to separate files!